src/libs/actions/IOU.ts

```diff
@@ -6836,7 +6836,7 @@ function canApproveIOU(
     const reportNameValuePairs = chatReportRNVP ?? ReportUtils.getReportNameValuePairs(iouReport?.reportID);
     const isArchivedReport = ReportUtils.isArchivedRoom(iouReport, reportNameValuePairs);
     const allViolations = violations ?? allTransactionViolations;
-    const hasViolations = ReportUtils.hasViolations(iouReport?.reportID ?? '-1', allViolations);
+    const hasNonHeldViolations = ReportUtils.hasNonHoldViolations(iouReport?.reportID ?? '-1', allViolations);
```

Comment:
Lets call it hold as in other places
```suggestion
    const hasNonHoldViolations = ReportUtils.hasNonHoldViolations(iouReport?.reportID ?? '-1', allViolations);
```